src/mscorlib/shared/System/IO./Path.Unix.cs

```diff
@@ -0,0 +1,218 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
that's absolutely fantastic. Windows doesn't show directories with a period at the end, not even in bash -,-